packages/wallet/readme.md

```diff
@@ -0,0 +1,11 @@
+#### 📚 [Documentation](https://docs.walletconnect.com/2.0/web3modal/about)
+
+#### 🔗 [Website](https://web3modal.com)
+
+# Web3Modal
+
+Your on-ramp to web3 multichain. Web3Modal is a versatile library that makes it super easy to connect users with your Dapp and start interacting with the blockchain.
```

Comment:
Besides of main `README.md` content, it might make sense to tell more about the package imo. What is it and why it's created etc. 

Not blocker but good to have